portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,9 +47,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     for (Entity entity : results.asIterable()) {
+      /* Convert the comment's timestamp (ms since Epoch) to a date (d/m/y) */
+      SimpleDateFormat sdf = new java.text.SimpleDateFormat("MMMM d, yyyy");
```

Comment:
It is better to pass the raw data (timestamp in ms since Epoch) back to the frontend and let the frontend decide how to format it. there are some javascript libraries that make this easier. For example https://momentjs.com/